e2e_tests/docker_image/parsec-service-test-all.Dockerfile

```diff
@@ -105,3 +105,33 @@ RUN useradd -m parsec-client-2
 
 # Add `/usr/local/lib` to library path for Trusted service provider
 ENV LD_LIBRARY_PATH="/usr/local/lib"
+
+# These commands are made to create keys and key mappings using an old version of Parsec.
+# One test will try to use those keys again to make sure they still work.
+RUN mkdir /tmp/create_keys
+RUN git clone https://github.com/parallaxsecond/parsec.git --branch 0.7.0 /tmp/create_keys/parsec \
+	&& cd /tmp/create_keys/parsec \
+	&& cargo build --features "all-providers, all-authenticators"
+ENV PARSEC_SERVICE_ENDPOINT="unix:/tmp/parsec.sock"
+RUN tpm_server & \
+    sleep 5 \
+    && tpm2_startup -c \
+    && tpm2_takeownership -o tpm_pass \
+    && cd /tmp/create_keys/parsec/e2e_tests \
+    && SLOT_NUMBER=`softhsm2-util --show-slots | head -n2 | tail -n1 | cut -d " " -f 2` \
+    && find . -name "*toml" -not -name "Cargo.toml" -exec sed -i "s/^# slot_number.*$/slot_number = $SLOT_NUMBER/" {} \; \
+    && cd ../ \
+    && ./target/debug/parsec -c e2e_tests/provider_cfg/all/config.toml & \
```

Comment:
I was lazy and I did not create the keys for the Trusted Service provider because it was not included in this config file. I will add it.

Reply:
Don't worry about it, I can do it in a different PR